recipes-bsp/u-boot/u-boot-imx_2017.03.bb

```diff
@@ -20,5 +20,29 @@ inherit dtc-145 fsl-u-boot-localversion
 
```

Comment:
Maybe the title should be "deploy boot image generation tools" as the patch doesn't deplay on "boot partition".